### ISSUE TITLE
Improve vendor chunk splits along routes

### ISSUE THREAD

**Issue by peterp** [redwoodjs/redwood#2071]
It came to our attention that we're not actually splitting vendor modules per-route.

Realated dicussion: https://community.redwoodjs.com/t/trouble-with-webpack-bundle-size-and-loading-performance/1949

**Comment by Irev-Dev** [comment]
Thanks @peterp.

You asked for a PR in the discussion, I'm not exactly sure what you want in the PR? Are you after a demo of code where the vendor bundles isn't split by route?

**Comment by dac09** [comment]
I'm looking into this further @Irev-Dev! What might be very useful to me is if you mentioned what libraries you're using so I can test it out.

**Comment by Irev-Dev** [comment]
My repo is here: https://github.com/Irev-Dev/cadhub @dac09 

I figure that's better than me listing libraries one by one here? or should I still do that?

The fact that I have [modified the webpack config](https://github.com/Irev-Dev/cadhub/blob/main/web/config/webpack.config.js) might complicate things.

I could probably spin up a new redwood projects that the minimum needed to show vendors not being split per route though?

**Comment by peterp** [comment]
Sorry @Irev-Dev,

> You asked for a PR in the discussion
I was on a call and typing at the same time (a bad habit), I meant to ask for an issue instead of a PR.

**Comment by dac09** [comment]
Hey @Irev-Dev - would you like to try the PR build in #2082 and let me know how it goes for you?

https://github.com/redwoodjs/redwood/pull/2082#issuecomment-805808178

**Comment by Irev-Dev** [comment]
Thanks @dac09 
I just tried the PR install myself and it didn't work either, but your `--stats` report looks really good.

**Comment by dac09** [comment]
Hey Kurt, now that the PR is merged, would you like to give it a try in canary? 

`yarn rw upgrade -t canary`

Let us know how you get on!

**Comment by Irev-Dev** [comment]
Really happy with this, I ran `yarn rw build web --stats` and it's definitely splits things, but then I changed a few imports to dynamic imports and I can see the effect of that too. Gives me a lot more confidence that I'll be able to manage my chunk sizes and control how and when js gets to the app 🙌 

Is this likely to make it into 0.29?

Here are two screen shots for good measure.
![image](https://user-images.githubusercontent.com/29681384/112604744-9c023500-8e6a-11eb-8334-4f3f009d6555.png)

![image](https://user-images.githubusercontent.com/29681384/112604775-a58b9d00-8e6a-11eb-995c-9206554f6f4e.png)
